src/TLuaInterpreterNetworking.cpp

```diff
@@ -625,8 +625,8 @@ int TLuaInterpreter::getHTTP(lua_State* L)
         }
     }
 
-    host.updateProxySettings(host.mLuaInterpreter.mpFileDownloader);
-    QNetworkReply* reply = host.mLuaInterpreter.mpFileDownloader->get(request);
+    host.updateProxySettings(host.getNetworkAccessManager());
```

Comment:
Since we are now using a QNAM from another class and not using our own do we actually need/want to `Host::updateProxySettings(...)` here and elsewhere - shouldn't we just do it once on creation of the QNAM and leave it unless it somehow needs to be changed?